scripts/rosversion

```diff
@@ -37,7 +37,13 @@ import rospkg
 from rospkg.common import PACKAGE_FILE
 from rospkg.rospack import ManifestManager
 
-USAGE = "Usage: rosversion <package/stack> or rosversion -d"
+USAGE = """Usage:
+rosversion <package/stack> [option]
+rosversion -d [option]
+
+Option:
+-n: Add no new line at the end of the output line.
```

Comment:
The help text is a bit difficult to read. I would suggest:

> Skip trailing new line

And based on that name the option `-s`.